<!DOCTYPE html>
<html style=""><head>
  <meta http-equiv="content-type" content="text/html; charset=UTF-8">
  <style type="text/css">
    .node {
    stroke: #fff;
    stroke-width: 1.5px;
}
.link {
    stroke: #999;
    stroke-opacity: .6;
}

h3 {
    color: #1ABC9C;
    text-align:center;  
    font-style: italic;
    font-size: 14px;
    font-family: "Helvetica";
}
  </style>

  <title>k-mer similarity, networks of microbial genomes and taxonomic rank</title>

<link href='http://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>
<link href='http://fonts.googleapis.com/css?family=Roboto:300' rel='stylesheet' type='text/css'>
<link href='http://fonts.googleapis.com/css?family=Droid+Sans+Mono' rel='stylesheet' type='text/css'>
<link rel="stylesheet" href="gg_styles.css" type="text/css">
  
</head>

<body>

<div id="main">

<div id="mainpage">
<p>
Supplementary Data Site
</p>

<span id="beauty">
<i>k</i>-mer similarity, networks of microbial genomes and taxonomic rank
</span>
<br>
<b><p>Guillaume Bernard<sup>1</sup>, Paul Greenfield<sup>2</sup>, Mark A. Ragan<sup>1</sup> and Cheong Xin Chan<sup>1,3</sup></p></b>

<p>
<sup>1</sup>Institute for Molecular Bioscience, The University of Queensland, Brisbane, QLD 4072, Australia<br>
<sup>2</sup>CSIRO, North Ryde, NSW 2113, Australia<br>
<sup>3</sup>School of Chemistry and Molecular Biosciences, The University of Queensland, Brisbane, QLD 4072, Australia<br>
</p>

<br>

<p>
<b>Dynamic Figure 2.</b> <i>P</i>-network of 2705 prokaryote genomes based on whole-genome data. Network generated using <i>D<sub>2</sub><sup>S</sup></i> with <i>k</i> = 25 at <i>t</i> = 3. This figure correponds to Figure 2 of this paper. [<a href="clustNet_bacteria_lim3.html" target="_blank">View</a>]
</p>

<p>
<b>Dynamic Figure 3.</b> <i>P</i>-network of 2616 prokaryote genomes based on chromosomal sequences with rRNA genes removed. Network generated using <i>D<sub>2</sub><sup>S</sup></i> with <i>k</i> = 25 at <i>t</i> = 3. This figure correponds to Figure 3 of this paper. [<a href="clustNet_noRNA_t3.html" target="_blank">View</a>]
</p>

<p>
<b>Dynamic Figure 4.</b> <i>I</i>-network of 921 plasmid genomes. Network generated using <i>D<sub>2</sub><sup>S</sup></i> with <i>k</i> = 25. Network at threshold <i>t</i> = 0 is shown in Figure 4 of this paper. [<a href="plasmid_t0.html" target="_blank">View</a>]
</p>


<br>

<p>
<b>All phylogenomic networks</b> generated from this work are available for download at <a href="https://dx.doi.org/10.14264/uql.2017.436" target="_blank">https://dx.doi.org/10.14264/uql.2017.436</a>. 
<li>Please see this <a href="readme.txt" target="_blank">README page</a> for the use of these downloaded files on your local computer.</li>
</p>

<br>
<hr>

<b>Citation</b><br>
Bernard G, Greenfield P, Ragan MA and Chan CX (2018) <i>k</i>-mer similarity, networks of microbial genomes and taxonomic rank. <i>mSystems</i> 3: e00257-18.<br>
doi:<a href="https://doi.org/10.1128/mSystems.00257-18" target="_blank">10.1128/mSystems.00257-18</a> | <a href="https://msystems.asm.org/content/3/6/e00257-18.full.pdf" target="_blank">PDF</a><br>

<br>
<b>Contact</b><br>
Cheong Xin Chan <span id="code"> | c.chan1 [at] uq.edu.au</span>
</p>

</div>
</div>